facturacion_mexico/cfdi_recibidos/services/xml_ingestion.py

```
@@ -0,0 +1,139 @@
+	    message: descripción del resultado
+	"""
+	if not xml_bytes:
+		return _result("error", None, None, "XML vacío")
+
+	xml_hash = _sha256(xml_bytes)
+
+	# Paso 1: parsear
+	try:
+		parser = CFDIRecibidoParser(xml_bytes)
+		data = parser.parse()
+	except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)